alvenir/guidance/v1/guidance.proto

```diff
@@ -1,67 +1,73 @@
 syntax = "proto3";
+
 package alvenir.guidance.v1;
 
 
 message LlmPromptAnswer {
-  string prompt = 1;
-  string context = 2;
-  string answer = 3;
+    string prompt = 1;
+    string context = 2;
+    string answer = 3;
 }
 
 message LlmData {
-  LlmPromptAnswer introduction = 1;
-  LlmPromptAnswer client_understanding = 2;
-  LlmPromptAnswer follow_up_questions = 3;
-  LlmPromptAnswer answer_quality = 4;
-  LlmPromptAnswer language = 5;
-  LlmPromptAnswer conclusion = 6;
-  LlmPromptAnswer follow_up = 7;
-  LlmPromptAnswer general_feedback = 8;
+    LlmPromptAnswer introduction = 1;
+    LlmPromptAnswer client_understanding = 2;
+    LlmPromptAnswer follow_up_questions = 3;
+    LlmPromptAnswer answer_quality = 4;
+    LlmPromptAnswer language = 5;
+    LlmPromptAnswer conclusion = 6;
+    LlmPromptAnswer follow_up = 7;
+    LlmPromptAnswer general_feedback = 8;
 }
 
 message CallMeta {
-  string time_of_call = 1;
-  string length = 2;
-  float answer_time = 3;
+    string time_of_call = 1;
+    string length = 2;
+    float answer_time = 3;
 }
 
 message TranscriptionMeta {
-  string n_words = 1;
-  string n_fill_words = 2;
+    string n_words = 1;
+    string n_fill_words = 2;
 }
 
 // for timeline chart
-message StringData {
-  repeated string x = 1;
-  repeated string y = 2;
-  string speaker = 3;
+message TimeInterval {
+    string start = 1;
+    string end = 2;
+}
```

Comment:
A lot of string values seem to describe numeric data - why are they strings?

Reply:
Fordi formatet i python bliver lavet om til HH:MM::ss:ms så dumme javascript forstår det

Reply:
Også n_words og n_fill_words? :)

Reply:
samme argument as above